testinfra/modules/pip.py

```diff
@@ -23,10 +24,37 @@ def _re_match(line, regexp):
     return match.groups()
 
 
-class PipPackage(InstanceModule):
-    """Test pip packages status and version"""
+class Pip(Module):
+    """Test pip package manager and packages"""
 
-    def check(self, pip_path="pip"):
+    def __init__(self, name, pip_path="pip"):
+        self.name = name
+        self.pip_path = pip_path
+        super().__init__()
+
+    @property
+    def is_installed(self):
+        """Test if the package is installed
+
+        >>> host.package("pip").is_installed
+        True
+        """
+        return self.run_test("{} show %s".format(self.pip_path), self.name).rc == 0
```

Comment:
I think `self.pip_path` should be eventually shell escaped (in case of spaces or shell restricted characters). This can be done in a followup PR or commit.

```
return self.run_test("%s show %s", self.pip_path, self.name).rc == 0
```

Reply:
Done for all calls.